modules/rntuple.mjs

```diff
@@ -155,15 +155,28 @@ function getTypeByteSize(coltype) {
         case ENTupleColumnType.kInt64:
         case ENTupleColumnType.kUInt64:
         case ENTupleColumnType.kIndex64:
+        case ENTupleColumnType.kSplitInt64:
+        case ENTupleColumnType.kSplitUInt64:
+        case ENTupleColumnType.kSplitReal64:
             return 8;
+
         case ENTupleColumnType.kReal32:
         case ENTupleColumnType.kInt32:
         case ENTupleColumnType.kIndex32:
         case ENTupleColumnType.kUInt32:
+        case ENTupleColumnType.kSplitInt32:
+        case ENTupleColumnType.kSplitUInt32:
+        case ENTupleColumnType.kSplitReal32:
+        case ENTupleColumnType.kSplitIndex32:
         case ENTupleColumnType.kSplitIndex64:
+        case ENTupleColumnType.kReal32Trunc:
+        case ENTupleColumnType.kReal32Quant:
             return 4;
```

Comment:
The Real32Trunc/Quant types should not be here - as they have variable bit width and we don't support them at the moment